packages/storage-web-api/get-website-scan-func/function.json

```diff
@@ -0,0 +1,18 @@
+{
+    "bindings": [
+        {
+            "authLevel": "anonymous",
+            "name": "request",
+            "type": "httpTrigger",
+            "direction": "in",
+            "route": "websites/{websiteId}/scans/{scanType}/{scanTarget}",
```

Comment:
probably the route should looks like `websites/{websiteId}/{scanType}/scans/{scanId}`
the `scanTarget` should be `scanId`

Reply:
the scan target can be either an id or "latest," depending on whether the user is requesting a specific scan or the latest scan of that type

Reply:
still `scanTarget` name is confusing a bit, maybe `scanIdOrLatest`